playwright/test/helpers/utils.ts

```diff
@@ -15,8 +15,15 @@ export type CookieType = {
 
 const togglePrefix = 'toggle_';
 
-export function makeDefaultToggleAndTestCookies(domain: string): CookieType[] {
-  return [...toggleConfig.toggles, ...toggleConfig.tests].map(t => {
+export async function makeDefaultToggleAndTestCookies(
+  domain: string
+): Promise<CookieType[]> {
+  const { data } = await axios.get(
+    'https://toggles.wellcomecollection.org/toggles.json'
+  );
```

Comment:
If this works, OK - but it is going to create an issue of PROD toggles being used.